README.md

```diff
@@ -1,2 +1,32 @@
 # full-stack-test-template
-A template to fork for our full-stack test
+
+A monorepo template to fork or use as reference for our full-stack test.
+
+There are also READMEs in each workspace, though they are just the default READMEs generated by NestJS & Vite.
+
+# Prerequisites
+- [nvm](https://github.com/nvm-sh/nvm?tab=readme-ov-file#installing-and-updating) or [Node.js v20.12.0](https://nodejs.org/en/download)
+- [pnpm](https://pnpm.io/installation#using-corepack) _(We recommend installing with Corepack)_
+- (Optional) [nx](https://nx.dev/getting-started/installation) for running workspaces
+
+# Technologies
+
+Just like the stack we use in our monorepo, this monorepo uses [**NestJS**](https://docs.nestjs.com/) for the API, and **React** (bundled by [**Vite**](https://vitejs.dev/guide/)) for the front end web app. We use **pnpm** for our package manager, which also manages the workspaces within the monorepo.
+
+# Getting Started
+
+After installing the prerequisites, you can use `nvm` to install & path the correct Node version based on the `.nvmrc` in this repository.
+
+```bash
+nvm install
+nvm use
+```
```

Comment:
Should we add a `pnpm i` line here, too?

Reply:
Yes we do